astropy/wcs/src/wcslib_celprm_wrap.c

```diff
@@ -169,6 +175,7 @@ static PyObject* PyCelprm_deepcopy(PyCelprm* self)
 
     memcpy(copy->x, self->x, sizeof(struct celprm));
     copy->x->err = NULL;
+    self->x->flag = 0;
```

Comment:
Apologies for a drive-by comment, should this be `copy->x` instead of `self->x`? (Basically, why would copying from a source change anything about the source but I can see that the destination may not be fully set)

Reply:
Oh, yes, this is a mistake.